deltacat/tests/compute/test_compact_partition_rebase_then_incremental.py

```diff
@@ -0,0 +1,347 @@
+
+    sort_keys = setup_sort_keys(sort_keys_param)
+    partition_keys = setup_partition_keys(partition_keys_param)
+    delta_type = DeltaType(input_deltas_delta_type)
+    (
+        source_table_stream,
+        destination_table_stream,
+        rebased_table_stream,
+    ) = create_table_strategy(
+        primary_keys_param,
+        sort_keys,
+        partition_keys,
+        column_names_param,
+        input_deltas_arrow_arrays_param,
+        delta_type,
+        partition_values_param,
+        ds_mock_kwargs,
+    )
+    source_partition: Partition = ds.get_partition(
+        source_table_stream.locator,
+        partition_values_param,
+        **ds_mock_kwargs,
+    )
+    destination_partition_locator: PartitionLocator = PartitionLocator.of(
+        destination_table_stream.locator,
+        partition_values_param,
+        None,
+    )
+    rebased_partition: Partition = ds.get_partition(
+        rebased_table_stream.locator,
+        partition_values_param,
+        **ds_mock_kwargs,
+    )
+    num_workers, worker_instance_cpu = DEFAULT_NUM_WORKERS, DEFAULT_WORKER_INSTANCE_CPUS
+    total_cpus = num_workers * worker_instance_cpu
+    pgm = None
+    create_placement_group_param = False
+    if create_placement_group_param:
+        pgm = PlacementGroupManager(
+            1, total_cpus, worker_instance_cpu, memory_per_bundle=4000000
+        ).pgs[0]
+    hash_bucket_count_param = None
+    records_per_compacted_file_param = DEFAULT_MAX_RECORDS_PER_FILE
+    compact_partition_params = CompactPartitionParams.of(
+        {
+            "compaction_artifact_s3_bucket": TEST_S3_RCF_BUCKET_NAME,
+            "compacted_file_content_type": ContentType.PARQUET,
+            "dd_max_parallelism_ratio": 1.0,
+            "deltacat_storage": ds,
+            "deltacat_storage_kwargs": ds_mock_kwargs,
+            "destination_partition_locator": destination_partition_locator,
+            "hash_bucket_count": hash_bucket_count_param,
+            "last_stream_position_to_compact": source_partition.stream_position,
+            "list_deltas_kwargs": {**ds_mock_kwargs, **{"equivalent_table_types": []}},
+            "pg_config": pgm,
+            "primary_keys": primary_keys_param,
+            "rebase_source_partition_locator": source_partition.locator,
+            "records_per_compacted_file": records_per_compacted_file_param,
+            "s3_client_kwargs": {},
+            "source_partition_locator": rebased_partition.locator,
+            "sort_keys": sort_keys if sort_keys else None,
+        }
+    )
+    # execute
+    rcf_file_s3_uri = compact_partition_func(compact_partition_params)
+    compacted_delta_locator = get_compacted_delta_locator_from_rcf(
+        setup_s3_resource, rcf_file_s3_uri
+    )
+    tables = ds.download_delta(compacted_delta_locator, **ds_mock_kwargs)
+    compacted_table = pa.concat_tables(tables)
+    assert compacted_table.equals(
+        rebase_expected_compact_partition_result
```

Comment:
Same here regarding combine_chunks and sort.

Reply:
Done